pallets/living-assets-ownership/src/traits.rs

```diff
@@ -92,3 +94,128 @@ impl AsRef<[u8]> for Erc721Error {
 		}
 	}
 }
+
+#[cfg(test)]
+mod tests {
+	use frame_support::{assert_err, assert_ok};
+
+	use super::*;
+	use crate::{mock::*, Event};
+
+	type AccountId = <Test as frame_system::Config>::AccountId;
+	const ALICE: AccountId = 0x1234;
+
+	#[test]
+	fn base_uri_of_unexistent_collection_is_none() {
+		new_test_ext().execute_with(|| {
+			assert_eq!(<LivingAssetsModule as CollectionManager<AccountId>>::base_uri(0), None);
```

Comment:
We are testing similar use cases in pallets/living-assets-ownership/src/tests.rs. Do you think it is worthwhile to maintain duplicated tests for both the trait and the specific type?

Reply:
I had in mind to retain only those related to the pallet and to remove the ones from the trait. Seeing that you've taken care of this, I'll mark the comment as resolved.